opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -604,6 +677,7 @@ def __init__(
         links: Sequence[trace_api.Link] = (),
         kind: trace_api.SpanKind = trace_api.SpanKind.INTERNAL,
         span_processor: SpanProcessor = SpanProcessor(),
+        limits: _Limits = _UnsetLimits,
```

Comment:
I understand that you are trying to make limits private with `_Limits`, but I think exposing it here as an attribute type and default argument defeats the intended purpose. This is putting a private object in a public place.

Reply:
Isn't `Span` an internal interface though? Otel API forbids anyone from creating spans directly so I don't see this as a public change but happy to change it if there are other use cases where we treat Span as part of public SDK interface.